src/common/filesystem.c

```diff
@@ -1614,6 +1616,27 @@ FS_GetNextRawPath(const char* lastRawPath)
 	return NULL;
 }
 
+#ifdef _MSC_VER // looks like MSVC/the Windows CRT doesn't have basename()
+// returns the last part of the given pathname, after last (back)slash
+// NOTE: this is not a fully compliant basename() implementation, as it doesn't
+//       handle trailing (back)slashes at all - because we don't need that.
+static char* basename( char* n )
```

Comment:
would `_splitpath` do the job ?

Reply:
nope, it cuts off the file extension

Reply:
yeah, using _splitpath (or more likely _splitpat_s) and then concatenating things again etc isn't really easier or less error-prone than this